README.md

```diff
@@ -19,14 +19,50 @@ go get gopkg.in/DataDog/dd-trace-go.v1/contrib/gorilla/mux
 
 If you installed more packages than you intended, you can use `go mod tidy` to remove any unused packages.
 
-Requires:
+### Documentation
+
+The API is documented on [godoc](https://godoc.org/gopkg.in/DataDog/dd-trace-go.v1/ddtrace) as well as Datadog's official documentation for [Tracing Go Applications](https://docs.datadoghq.com/tracing/setup/go/) and the [Continuous Go Profiler](https://docs.datadoghq.com/tracing/profiler/enabling/go). If you are migrating from an older version of the tracer (e.g. 0.6.x) you may also find the [migration document](https://github.com/DataDog/dd-trace-go/blob/v1/MIGRATING.md) we've put together helpful.
+
+### Support Policy
+
+Datadog APM for go is built upon dependencies defined in specific versions of the host operating system, go releases, and the Datadog Agent/API. For go the two latest releases are [GA](#support-ga) supported and the version before that is in [Maintenance](#support-maintenance). We do make efforts to support older releases, but generally these releases are considered [Legacy](#support-legacy). This library only officially supports [first class ports](https://github.com/golang/go/wiki/PortingPolicy#first-class-ports) of go.
+
+| **Level**                                              | **Support provided**                                                                                                                                                          |
+|--------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| <span id="support-ga">General Availability (GA)</span> |  Full implementation of all features. Full support for new features, bug & security fixes.                                                                                    |
+| <span id="support-maintenance">Maintenance</span>      |  Full implementation of existing features. Does not receive new features. Support for bug & security fixes only.                                                              |
```

Comment:
Just wondering here if we should change "Does not receive new features" to "May not receive new features".
I'm thinking users using a version of Go in "Maintenance" will see new features and assume that they're in a "GA" version, and get surprised when they are suddenly in "Legacy" and there's no support at all.

I may be overthinking this, though.

Reply:
I'm definitely fine with moving to "May not receive new features" to make it clearer that new features might arrive for those versions but aren't guaranteed. You might want to also see this comment https://github.com/DataDog/dd-trace-go/pull/1239#discussion_r844201852 about possibly removing the distinction of "maintenance" entirely